staker-rewards/rewards.js

```diff
@@ -122,7 +140,7 @@ run()
     process.exit(0)
   })
   .catch((error) => {
-    console.error(
+    console.trace(
```

Comment:
Why are we changing the log level here?

Reply:
Outputting a stack trace may help on errors investigation. I wanted to commit it in a separate commit but it got included accidentialy.

Reply:
Yeah, lesson learned here as well. It's not a log level.